src/Api/Endpoints/Reporting.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Passwordless.Api.Authorization;
+using Passwordless.Common.Models.Reporting;
+using Passwordless.Service;
+using static Microsoft.AspNetCore.Http.Results;
+
+namespace Passwordless.Api.Endpoints;
+
+public static class ReportingEndpoints
+{
+    public static void MapReportingEndpoints(this WebApplication app)
+    {
+        var group = app.MapGroup("/apps/{appId}/reporting")
```

Comment:
Add note about this being an internal endpoint (should not be documented or expected to be stable)

Reply:
Absolutely, but we should do a proper API review before making it public since it's hard to change it afterwards.